Gemfile

```diff
@@ -1,9 +1,11 @@
 source "https://rubygems.org"
 
-gem 'bosh_cli', '1.3262.26.0'
+gem 'aws-sdk', '~> 2.3.8'
+gem 'cf-uaa-lib', '~> 3.8.0'
```

Comment:
This failed to add cf-uaac that was in the previous Gemfile. This is needed for interacting with UAA (as documented here: https://government-paas-team-manual.readthedocs.io/en/latest/guides/CF_UAA_user_management/)